### PR TITLE
Add configuration `change-theme`

### PR DESCRIPTION
**Pull request by flamingoodev**
Add configuration `change-theme` for `Change theme` button.

### CODE REVIEW

File: src/args.rs [hunk 1]
```diff
@@ -83,6 +83,10 @@ pub struct CliArgs {
     #[clap(short = 'H', long = "hidden")]
     pub hidden: bool,
 
+    /// Enable change theme display
```
Comment (svenstaro) [inline src/args.rs:86]: How about
```suggestion
    /// Show theme selector in frontend
```
?

File: src/config.rs [hunk 2]
```diff
@@ -81,6 +81,9 @@ pub struct MiniserveConfig {
     /// Enable QR code display
     pub show_qrcode: bool,
 
+    /// Enable change theme display
```
Comment (svenstaro) [inline src/config.rs:84]: How about
```suggestion
    /// Show theme selector in frontend
```
?

**Review by svenstaro — CHANGES_REQUESTED** [review]
Seems fair enough. How about changing the text slightly?

File: src/args.rs [hunk 3]
```diff
@@ -83,6 +83,10 @@ pub struct CliArgs {
...
+    #[clap(short = 'c', long = "change-theme")]
```
Comment (svenstaro) [inline src/args.rs:87]: Also this can't work since now you use `-c` twice. Does this need a short option?

**Review by svenstaro — CHANGES_REQUESTED** [review]

File: src/config.rs (hunk 2 above)
Comment (flamingoodev) [reply src/config.rs:84]: Sounds good, thanks for the suggestion.

File: src/args.rs (hunk 3 above)
Comment (flamingoodev) [reply src/args.rs:87]: Oh,I didn't notice it was used,I'll remove it.

File: src/args.rs (hunk 1 above)
Comment (svenstaro) [reply src/args.rs:86]: This still isn't fixed.

**Review by svenstaro — CHANGES_REQUESTED** [review]
You only applied one of the suggestions.

**Comment by svenstaro** [comment]
Thanks!

**Pull request merged**